### ISSUE TITLE
Applying extra col-*-* classes to columns within the editor breaks things

### ISSUE THREAD

**Issue by Mike-Morris** [madoublet/respond#293]
Quite a subtle bug, and often doesn't have any noticeable effect in the browser which I had seen quite some time ago but assumed was down to shoe-horning a premade theme into Respond, however... Using the vanilla Bootstrap template included with respond 4.8(dev), i've noticed the same thing still happening.

Basically, if I apply any col-*-* class to a column within the editor, it seems to add repeated/extra classes  upon saving the output HTML. - I see a lot of 'sortable sortable sortable sortable' in the class field.

Any chance this is an easy fix? - As respond uses the col-md options to define how the editor will look, perhaps be passive about any XS/SM/LG classes applied?

**Comment by Mike-Morris** [comment]
![screen shot 2015-05-11 at 18 20 34](https://cloud.githubusercontent.com/assets/2870589/7570821/fabff1e2-f80a-11e4-859f-b96b1a1f6669.png)

**Comment by madoublet** [comment]
For some reason, I don't have this issue.  I am not sure what is causing it.  What browser are you using?

**Comment by amit2602** [comment]
Hi Matt, I also have this problem, even with rows that doesnt have col-* .. regular rows and blocks gets this "sortable" class repeated everytime I save the page. Until its too much and the page is breaking.

**Comment by madoublet** [comment]
@amit2602 Is this happening in Chrome?  Or, are you using Firefox?  I am trying to get to the bottom of it.  If you view the source on respondcms.com (and the interior pages), this is not happening.  And, I edit this site a lot.

**Comment by Mike-Morris** [comment]
Yes using chrome pretty much exclusively.

Not sure if the trigger is using multiple classes in the class field, or what the value of those classes is, but something along the line is causing this to happen, even if I don't update a particular area in the editor and simply save it, I end up with a bigger and bigger accumulation of these classes as per the picture.

The place this really becomes a problem is when it causes the page become larger than MySQL's size limit for the field.

**Comment by Mike-Morris** [comment]
Could also be caused if you are splitting the field by spaces, but there happens to be two spaces together? Just a guess :/

**Comment by amit2602** [comment]
@madoublet Yes I'm using chrome.
Also, I tried to increase the size limit for pages content, so I changed the type of the 'content' column in 'pages' table to 'mediumtext', but for some reason after I edited some page it didnt display the changes in the page, but the 'content' filed for the page WAS updated... Is this got something to do with how respond reads the fields from the database? if so, is there an option to add 'textmedium' type along with the default 'text' type fields?

Thanks alot Matt! :+1:

**Comment by madoublet** [comment]
@amit2602 I have no idea what changing the type does.  I am not sure what effect that would have.

@Mike-Morris I am still not able to replicate it.  Is the site brand new or did it come from an older version of Respond?

**Comment by amit2602** [comment]
@madoublet the type is setting the size limit of the field, check this out:
http://www.electrictoolbox.com/maximum-length-mysql-text-field-types/
So even there, they are suggesting to use the 'mediumtext' if you are storing web pages in a field.
The default 'text' isnt enough, long pages are breaking.

**Comment by amit2602** [comment]
@madoublet, for me its happening on both new and old sites. this is a new site from yesterday:
http://screencast.com/t/iErMXUYqYcQV
this is terrible, I didnt even save that much times...

**Comment by madoublet** [comment]
Ha, yeah, that is pretty bad.  I can't reproduce it, but I think I can add a filter to prevent it.

**Comment by Mike-Morris** [comment]
New respond, using the basic bootstrap theme.

I didn't report it earlier because that wasn't the case, I had loaded a tonne of unnecessary JS and CSS into the template to clone a themeforest template, so thought that might have caused it.

It's a subtle one, but it's pretty severe for those of us getting it!

**Comment by madoublet** [comment]
I pushed an update to dev that I think will fix it.  I was working on a bunch of other changes as well and did not want to branch just for this fix.  So, it is kind of lumped together right now with the other changes that I pushed tonight.  It is not production ready yet (still needs localization and testing), but if you are in a non-production environment and want to check it out, take a look.  If not, I should have everything worked out in a few days.
